platform-coop-toolkit/coop-map-directory-index
# feat: add license model

## Description

Adds "license type" field to tools.

## Steps to test

1. Edit a tool.

**Expected behavior:** "License type" and "Free / libre / open source license" fields both appear.

## Additional information

Not applicable.

## Related issues

Resolves #61.
